src/ray/common/bundle_spec.cc

```diff
@@ -80,6 +80,13 @@ std::string FormatPlacementGroupResource(const std::string &original_resource_na
       original_resource_name, bundle_spec.PlacementGroupId(), bundle_spec.Index());
 }
 
+std::string FormatPlacementGroupImplicityResource(const PlacementGroupID &group_id,
+                                                  int64_t bundle_index) {
+  std::string str =
```

Comment:
I don't think we need bundle index for this resource?

Reply:
I am confused with this, I think the core issue we solved is user create actor with a specified placement(id) and  placement group bundle index, but no any resource specified, if no bundle index here, how do we know which bundle this actor should be scheduled to ?

Reply:
In current case, the two placement group resource label will only add when actor's resource is specified(this is the key)